CHANGELOG.md

```diff
@@ -22,7 +22,7 @@ All notable changes to iroh will be documented in this file.
 ### 🚜 Refactor
 
 - *(iroh)* Make use of quic-rpc-derive macros to prettify the rpc declarations ([#2508](https://github.com/n0-computer/iroh/issues/2508)) - ([026baaa](https://github.com/n0-computer/iroh/commit/026baaafdf9bd89feef8db4bfedfe9d9853551f3))
-- *(iroh-net)* [**breaking**] Move relay implemention in `iroh-net` behind `iroh-relay` cfg flag ([#2516](https://github.com/n0-computer/iroh/issues/2516)) - ([f37d9f9](https://github.com/n0-computer/iroh/commit/f37d9f9f339dd0de50b35a405ce203c36272a0b4))
```

Comment:
please revert the changes in here, 
this file needs to be ignored, as it is regenerated on release based on commit messages